lib/matplotlib/tests/test_dates.py

```diff
@@ -551,15 +552,16 @@ def test_concise_formatter_show_offset(t_delta, expected):
     (datetime.timedelta(weeks=52 * 200),
      ['$\\mathdefault{%d}$' % (t, ) for t in range(1980, 2201, 20)]),
     (datetime.timedelta(days=40),
-     ['$\\mathdefault{Jan}$', '$\\mathdefault{05}$', '$\\mathdefault{09}$',
-      '$\\mathdefault{13}$', '$\\mathdefault{17}$', '$\\mathdefault{21}$',
-      '$\\mathdefault{25}$', '$\\mathdefault{29}$', '$\\mathdefault{Feb}$',
-      '$\\mathdefault{05}$', '$\\mathdefault{09}$']),
+     ['$\\mathdefault{}$Jan$\\mathdefault{}$', '$\\mathdefault{05}$',
```

Comment:
is it possible to strip the empty `$\\mathdefault{}$` text?

Reply:
Yes, I remove this in the new commit.